docs/reference/search/request/sort.asciidoc

```diff
@@ -127,6 +127,9 @@ field support has a `nested` sort option with the following properties:
     should match with in order for its field values to be taken into account
     by sorting. Common case is to repeat the query / filter inside the
     nested filter or query. By default no `nested_filter` is active.
+`max_children`::
```

Comment:
I think something like this would be enough:
`The maximum number of children to consider per root document when picking the sort value.
Defaults to unlimited.`